plugins/services/src/js/containers/pod-instances/PodInstancesTable.js

```diff
@@ -443,6 +464,34 @@ class PodInstancesTable extends React.Component {
     );
   }
 
+  renderRegion(prop, instance, rowOptions) {
+    if (rowOptions.isParent) {
```

Comment:
One question out of interest, is there a reason you use this inverse guard clause pattern here?
I am used to seeing early returns in cases like this, e.g.

```js
if (!rowOptions.isParent) {
  return null;
}

return (...)
```

Reply:
you are right I was just misguided.

Reply:
Would be nice to have in the rest too @Poltergeist

Reply:
I have addressed the issues from @DanielMSchmidt unfortunately forgot to push them.